src/components/annotations/annotation_defaults.js

```diff
@@ -120,6 +120,7 @@ module.exports = function handleAnnotationDefaults(annIn, annOut, fullLayout, op
             color: hoverBorder
         });
     }
+    coerce('captureevents', !!hoverText);
```

Comment:
For annotations, `hovertext` is a string, I think this is OK. But you're absolutely right when this moves into traces and can be an array.